packages/manifold/contracts/gachaclaims/GachaLazyClaim.sol

```diff
@@ -0,0 +1,80 @@
+// SPDX-License-Identifier: MIT
+// solhint-disable reason-string
+pragma solidity ^0.8.0;
+
+import "@manifoldxyz/libraries-solidity/contracts/access/AdminControl.sol";
+
+import "./IGachaLazyClaim.sol";
+
+/**
+ * @title Gacha Lazy Claim
+ * @author manifold.xyz
+ */
+abstract contract GachaLazyClaim is IGachaLazyClaim, AdminControl {
+  using EnumerableSet for EnumerableSet.AddressSet;
+
+  string internal constant ARWEAVE_PREFIX = "https://arweave.net/";
+  string internal constant IPFS_PREFIX = "ipfs://";
+  address internal _signer;
+
+  uint256 internal constant MAX_UINT_8 = 0xff;
+  uint256 internal constant MAX_UINT_32 = 0xffffffff;
+  uint256 internal constant MAX_UINT_48 = 0xffffffffffff;
+  uint256 internal constant MAX_UINT_56 = 0xffffffffffffff;
+  uint256 internal constant MAX_UINT_80 = 0xffffffffffffffffff;
```

Comment:
i think this one is wrong -- the actual value is MAX_UINT_72 instead of MAX_UINT_80
```suggestion
  uint256 internal constant MAX_UINT_80 = 0xffffffffffffffffffff;
```